### PR TITLE
PORTALS-2478, PORTALS-2481, PORTALS-2482, PORTALS-2490, PORTALS-2497

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: apps/SageAccountWeb/src/style/_Navbar.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .top-nav {
-  background: rgba(256, 256, 256, 0.9);
+  background: rgba(256, 256, 256, 1);
```
Comment (jay-hodgson) [inline apps/SageAccountWeb/src/style/_Navbar.scss:2]: https://sagebionetworks.jira.com/browse/PORTALS-2481

File: apps/portals/src/_Core.scss [hunk 2]
```diff
@@ -281,6 +281,7 @@ a.nav-button {
 .nav-button-container.isSelected {
   @extend %hoverFade;
   @extend %bottomBorderNav;
+  border-radius: 0;
```
Comment (jay-hodgson) [inline apps/portals/src/_Core.scss:284]: https://sagebionetworks.jira.com/browse/PORTALS-2490

File: apps/portals/src/_Navbar.scss [hunk 3]
```diff
@@ -20,7 +20,7 @@
   display: none;
 }
 
-@media screen and (max-width: 768px) {
+@media screen and (max-width: 1280px) {
```
Comment (jay-hodgson) [inline apps/portals/src/_Navbar.scss:23]: https://sagebionetworks.jira.com/browse/PORTALS-2482

File: packages/synapse-react-client/src/lib/containers/MarkdownCollapse.tsx [hunk 4]
```diff
@@ -55,9 +55,14 @@ const MarkdownCollapse = (props: MarkdownCollapseProps) => {
...
-                navigator.clipboard.writeText(plainText).then(() => {
-                  displayToast('Successfully copied to the clipboard')
-                })
+                navigator.clipboard
+                  .writeText(plainText)
+                  .then(() => {
+                    displayToast('Successfully copied to the clipboard')
+                  })
+                  .catch(err => {
+                    console.error(err)
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/lib/containers/MarkdownCollapse.tsx:64]: To help investigate https://sagebionetworks.jira.com/browse/PORTALS-2510

File: packages/synapse-react-client/src/lib/containers/table/TablePagination.tsx [hunk 5]
```diff
@@ -46,6 +46,22 @@ export const TablePagination = () => {
         <option value={100}>100 per page</option>
         <option value={500}>500 per page</option>
       </select>
+      {
+        //TODO: PORTALS-2546: convert to MUI?
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/lib/containers/table/TablePagination.tsx:50]: Progress for https://sagebionetworks.jira.com/browse/PORTALS-2546

File: packages/synapse-react-client/src/lib/style/components/query_filter/_enum-facet-filter.scss [hunk 6]
```diff
@@ -26,11 +26,6 @@
       .checkbox,
       .checkbox-focused {
         margin: 5px 0px;
-        > label::before {
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/lib/style/components/query_filter/_enum-facet-filter.scss:29]: https://sagebionetworks.jira.com/browse/PORTALS-2497

File: packages/synapse-react-client/src/lib/style/components/_feed-cards.scss [hunk 7]
```diff
@@ -4,7 +4,11 @@
   font-weight: bold;
 }
 .FeedSubscribe {
+  display: flex;
+  justify-content: center;
   .FeedSubscribeToNewsLink {
+    display: flex;
+    align-items: center;
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/lib/style/components/_feed-cards.scss:11]: https://sagebionetworks.jira.com/browse/PORTALS-2478

**Review by hallieswan — APPROVED** [review]
Looks great!

**Pull request merged**